### ISSUE TITLE
Timeout error message should be a bytes string

### ISSUE THREAD

**Issue by JoshData** [gitpython-developers/GitPython#625]
At https://github.com/gitpython-developers/GitPython/blob/master/git/cmd.py#L650, I think the error message about timeouts should be a `b"..."` bytes string and not a regular string. You can see right afterwards in the endswith call that the string is expected to be a bytes string.

````
                        stderr_value = 'Timeout: the command "%s" did not complete in %d ' \
                                       'secs.' % (" ".join(command), kill_after_timeout)
                ....
                if stderr_value.endswith(b"\n"):
                    stderr_value = stderr_value[:-1]
````

So it should be:

````
                        stderr_value = b'Timeout: the command "%s" did not complete in %d ' \
                                       b'secs.' % (" ".join(command), kill_after_timeout)
````

**Comment by Byron** [comment]
@JoshData Thanks for letting me know. Would you like to contribute the fix?

**Comment by JoshData** [comment]
Yes will do.

**Comment by JoshData** [comment]
This was subsequently fixed by #682 in 1dbfd290609fe43ca7d94e06cea0d60333343838. Thanks!
